content/ru/_solutions/mediakiosk.md

```
@@ -0,0 +1,155 @@
+
+В ассортименте компании Wiren Board есть устройства для решения как простых, так и сложных задач автоматизации и диспетчеризации: программируемые логические контроллеры, датчики климата, счетчики электроэнергии, сетевые карты для холодильников, мощные релейные модули и т.д. Все это позволяет подстроиться под любые требования заказчика и использовать оборудование Wiren Board в его экосистеме. При этом обеспечивается возможность расширения экосистемы в будущем.
+
+Компания Wiren Board даёт на всё оборудование двухлетнюю гарантию с заменой оборудования без предварительной отправки в сервисный центр, а инженеры технической поддержки готовы помочь в решении как типовых, так и нестандартных задач.
+
+## Дополнительные материалы
+
+- [Сделано на Wiren Board: умный медиакиоск для остановок](https://habr.com/ru/companies/wirenboard/articles/848940/)
+
+Прочее
+
+- [Примеры других внедрений](../solutions/)
```

Markdownlint (0.35.0)

Expected: asterisk; Actual: dash
Unordered list style

(MD004, ul-style)